packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/components/ExpertOptions/ExpertOptionsContainer.spec.js

```diff
@@ -1,90 +0,0 @@
-import { shallow } from 'enzyme'
-import ExpertOptionsContainer from './ExpertOptionsContainer'
-import sinon from 'sinon'
-
-describe('TaskNavButtons > Component > ExpertOptionsContainer', function () {
-  let wrapper
-
-  describe('smoke test', function () {
-    before(function () {
-      wrapper = shallow(
-        <ExpertOptionsContainer />
-      )
-    })
-
-    it('should render without crashing', function () {
-      expect(wrapper).to.be.ok()
-    })
-  })
-
-  describe('when not in demo mode is not enabled', function () {
-    before(function () {
-      wrapper = shallow(
-        <ExpertOptionsContainer />
-      )
-    })
-
-    it('should render null', function () {
-      expect(wrapper).to.be.empty()
-    })
-  })
-
-  describe('when in demo mode is enabled by query param', function () {
-    let originalWindow
-    let setDemoModeSpy
-
-    before(function () {
-      setDemoModeSpy = sinon.spy()
-      originalWindow = window
-      global.window = {
-        ...window,
-        location: {
-          ...window.location,
-          search: '?demo=true'
-        }
-      }
-    })
```

Comment:
I could not for the life of me get this to work with RTL, so there is no unit test for whether ExpertOptionsContainer shows up when `demo=true` is in the url. This is an internal-facing feature, so I'm not worried about it.